tests/resources/promise-rejection-events.js

```diff
@@ -642,12 +844,12 @@ function onUnhandledSucceed(t, expectedReason, expectedPromiseGetter) {
 function onUnhandledFail(t, expectedPromiseGetter) {
   var unhandled = function(evt) {
     if (evt.promise === expectedPromiseGetter()) {
-      t.unreached_func('unhandledrejection event is not supposed to be triggered');
+      t.assert_unreached('unhandledrejection event is not supposed to be triggered');
```

Comment:
actually, I now get this error:

Harness Error. harness_status.status = 1 , harness_status.message = Uncaught TypeError: t.assert_unreached is not a function